### PR TITLE
Add ability to encrypt notes

### PR DESCRIPTION
**Pull request by yousefa00**
## Summary

Optionally, notes are encrypted using AES 256-bit encryption. Ideally, this should deter looking through the shared Atlas / snooping interview notes in the API, while allowing for continued ease of development without a mockdb and not allowing anyone (even a director) complete access to notes.

## Screenshots

Screen Shot 2021-05-09 at 5 00 31 AM
Screen Shot 2021-05-09 at 5 02 09 AM

### CODE REVIEW

**Review by daniel-moon32 — APPROVED** [review]
Looks good!

File: api/src/utils/notes.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+// decrypt a note with AES 256 encryption
+const decryptNote = async (encryptedNote, encryptionPassword) => {
+  const encryptedMessage = await openpgp.readMessage({
+    armoredMessage: encryptedNote, // parse encrypted bytes
+  });
+  const { data: decrypted } = await openpgp.decrypt({
+    message: encryptedMessage,
+    passwords: [encryptionPassword], // decrypt with password
+  });
+  console.log(decrypted);
```
Comment (Leundai) [inline api/src/utils/notes.js:34]: Should this be included?

File: client/src/pages/Note.js [hunk 2]
```diff
@@ -147,6 +146,8 @@ function Note({ user }) {
           setReferencedMembers(currentReferencedMembers.map((m) => m.memberId));
           setViewableBy(currentViewableBy.map((m) => m.memberId));
           setEditableBy(currentEditableBy.map((m) => m.memberId));
+          console.log(encrypt);
```
Comment (Leundai) [inline client/src/pages/Note.js:149]: this should be removed

**Review by Leundai — APPROVED** [review]

**Comment by Leundai** [comment]
Good stuff! :)

**Pull request merged**